### ISSUE TITLE
Angular ng command not recognized Error 

### ISSUE THREAD

**Issue by fussionlab** [pnp/generator-spfx#224]
#### General Information
* [ ] Usage
* [x] Development
* [ ] Documentation
* [ ] Feature Request

#### Specific generator
* [ ] Addons
* [ ] HandlebarsJS
* [x] Angular Element

#### What Version you are running?
* NodeJS: 10.16
* NPM: 6.13
* @pnp/spfx: 1.10.2
Getting an error that: 'ng' is not recognised as an internal or external command. While creating webpart. But I checked in the same terminal `ng --version` screenshot as below
![image](https://user-images.githubusercontent.com/23405898/72225661-0a761280-3599-11ea-8094-9192cad5f85a.png)
After creating Webpart it throws an error that above mentioned `ng is not recognised  command` screenshot below
![image](https://user-images.githubusercontent.com/23405898/72225784-7147fb80-359a-11ea-8526-37f232a7f56a.png)

![image](https://user-images.githubusercontent.com/23405898/72225738-f7b00d80-3599-11ea-8c7e-76908886c7aa.png)

But if I rerun `yo @pnp/spfx` it not creating angular element.

**Comment by StfBauer** [comment]
Seems like a problem with your Angular installation.

> 'ng' is not recognized as an internal or external command.

Means that the Angular cli cannot be found.
Also please check https://github.com/angular/angular-cli/issues/11722

**Comment by fussionlab** [comment]
> Seems like a problem with your Angular installation.
> 
> > 'ng' is not recognized as an internal or external command.
> 
> Means that the Angular cli cannot be found.
> Also please check [angular/angular-cli#11722](https://github.com/angular/angular-cli/issues/11722)

I got a few dought when we install pnp and angular both are globally installed right. In my case i can create a sperate angular app `ng new someapp`. it just working fine. 
*Above Error* when I use `yo @pnp/spfx` it's going fine in between it cause an error `ng`  is not recognized as an internal or external command.

**Comment by StfBauer** [comment]
@fussionlab Can you please check certain things for me on your machine, please?

**Check 1:** In PowerShell/CMD:
```shell
where ng
```

The output should be something link this:
```
C:\Users\stfB\AppData\Roaming\npm\ng
C:\Users\stfB\AppData\Roaming\npm\ng.cmd
```

**Check 2:** Location of Yo:

```shell
where yo
```

The output should be something like this:

```shell
C:\Users\stfB\AppData\Roaming\npm\ng
C:\Users\stfB\AppData\Roaming\npm\ng.cmd
```

**Check 3:** Overall installed packages of CLI and generator

```shell
npm ls -g @angular/cli @pnp/generator-spfx
```

The output should be something like this:

```shell
C:\Users\stfB\AppData\Roaming\npm
+-- @angular/cli@8.3.22
`-- @pnp/generator-spfx@1.10.2
```

Thank you very much.

**Comment by fussionlab** [comment]
> @fussionlab Can you please check certain things for me on your machine, please?
> 
> **Check 1:** In PowerShell/CMD:
> 
> ```shell
> where ng
> ```
> 
> The output should be something link this:
> 
> ```
> C:\Users\stfB\AppData\Roaming\npm\ng
> C:\Users\stfB\AppData\Roaming\npm\ng.cmd
> ```
> 
> **Check 2:** Location of Yo:
> 
> ```shell
> where yo
> ```
> 
> The output should be something like this:
> 
> ```shell
> C:\Users\stfB\AppData\Roaming\npm\ng
> C:\Users\stfB\AppData\Roaming\npm\ng.cmd
> ```
> 
> **Check 3:** Overall installed packages of CLI and generator
> 
> ```shell
> npm ls -g @angular/cli @pnp/generator-spfx
> ```
> 
> The output should be something like this:
> 
> ```shell
> C:\Users\stfB\AppData\Roaming\npm
> +-- @angular/cli@8.3.22
> `-- @pnp/generator-spfx@1.10.2
> ```
> 
> Thank you very much.

I just deleted everything and reinstalled. it's well and well installed all pack.
![image](https://user-images.githubusercontent.com/23405898/72276755-be32dd00-3641-11ea-9975-f51cb708e9cc.png)
but now the issue is #218 #222   `param-case` do exist in path
![image](https://user-images.githubusercontent.com/23405898/72277409-eec74680-3642-11ea-97e0-c873766542d3.png)

Let me try some possible way to override this issue.

**Comment by StfBauer** [comment]
@fussionlab Great - thank you very much for the update. I will look into it as well. It's a pretty strange issue in this case.

**Comment by fussionlab** [comment]
> @fussionlab Great - thank you very much for the update. I will look into it as well. It's a pretty strange issue in this case.

@StfBauer this was caused due to ```npm``` conflict.  Now it's fine so far, then Fixed the #222 Error. it quite running well.

**Comment by StfBauer** [comment]
@fussionlab thank you for the update.
